test/features/steps/step_bootstrap.py

```diff
@@ -0,0 +1,108 @@
+import re
+import logging
+from behave import given, when, then
+from crmsh import utils, bootstrap, corosync, parallax
+
+
+def run_command(context, cmd):
+    rc, out, err = utils.get_stdout_stderr(cmd)
+    if rc != 0 and err:
+        context.logger.error("{}\n".format(err))
+        context.failed = True
+    return out
+
+
+def check_cluster_state(context, state, addr):
+    if state not in ["started", "stopped"]:
+        context.logger.error("Cluster service state should be \"started/stopped\"\n")
+        context.failed = True
+
+    state_dict = {"started": True, "stopped": False}
+
+    if addr == "local":
+        assert bootstrap.service_is_active('pacemaker.service') is state_dict[state]
+    else:
+        test_active = "systemctl -q is-active pacemaker.service"
+        try:
+            parallax.parallax_call([addr], test_active)
+        except ValueError:
+            assert state_dict[state] is False
+        else:
+            assert state_dict[state] is True
+
+
+def online(context, nodelist):
+    rc = True
+    _, out = utils.get_stdout("crm_node -l")
+    for node in nodelist.split():
+        node_info = "{} member".format(node)
+        if not node_info in out:
+            rc = False
+            context.logger.error("Node \"{}\" not online\n".format(node))
+    return rc
+
+
+@given('Cluster service is "{state}" on "{addr}"')
```

Comment:
I think its quite strange that you put all the variables in quotes. Why is that convention? 
I wouldn't use them, it reduces human readibility. But if you think that it helps to identity variable I don't really care to keep them.

Reply:
variables without quotes like `'Cluster service is {state} on {addr}'` will case error in behave

Reply:
mmm, when I used behave in the past it wasn't this constraint...
Anyway, we can alwayas have a look on that in the future if we need